gotenv.go

```diff
@@ -174,9 +174,37 @@ func Write(env Env, filename string) error {
 	return file.Sync()
 }
 
+// splitLines is a valid SplitFunc for a bufio.Scanner. It will split lines on CR ('\r'), LF ('\n') or CRLF (any of the three sequences).
+// If a CR is immediately followed by a LF, it is treated as a CRLF (one single line break).
+func splitLines(data []byte, atEOF bool) (advance int, token []byte, err error) {
+	if atEOF && len(data) == 0 {
+		return 0, nil, bufio.ErrFinalToken
+	}
+
+	idx := bytes.IndexAny(data, "\r\n")
```

Comment:
I don't think `bytes` is being imported.

Reply:
oops, fixed